plugins/module_utils/net_map/networking_definition.py

```diff
@@ -129,6 +129,36 @@ def _validate_parse_field_type(
     return raw_value
 
 
+def _validate_parse_trunk_parent_field(
+    field_name: str,
+    raw_definition: typing.Dict[str, typing.Any],
+    parent_name: str = None,
+    parent_type: str = None,
+    trunk_parents: set = set(),
+) -> typing.Any:
+    trunk_parent = _validate_parse_field_type(
+        field_name,
+        raw_definition,
+        expected_type=str,
+        parent_name=parent_name,
+        parent_type=parent_type,
+        mandatory=False,
+    )
+    if trunk_parent and trunk_parent not in trunk_parents:
+        raise exceptions.NetworkMappingTrunkParentValidationError(
```

Comment:
Do we have a test that checks this condition is working?

Reply:
In almost all the tests based on the stubs we would evaluate `trunk_parent` to `None`.
I added two more tests in `test_networking_definitions_group_template.py` and `test_networking_definitions_instance.py`. 

I think that covers it?

Reply:
That's more than enough, thanks!